.github/workflows/pr.yml

```diff
@@ -10,11 +10,9 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
+        # "prepare" hook on `bun install` runs `bun run build`
```

Comment:
This can be defined with `nx.json` per [lerna docs](https://lerna.js.org/docs/concepts/task-pipeline-configuration#define-task-dependencies-aka-task-pipelines)

I can add that in separate PR - it doesn't seem to be an issue currently but it may be as the repo scales

Reply:
Filed as #32